website/docs/contributing.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 3
+---
+
+# How to contribute to FerretDB
+
+FerretDB is an open source project and everyone – developers and non-developers – is welcome to contribute.
+If you’re interested in contributing to FerretDB projects, this documentation will help you get started.
+
+Here are some of the current FerretDB projects that you can fork and contribute to:
+
+* [FerretDB](https://github.com/FerretDB/FerretDB): This repository contains the FerretDB project – an open source alternative project to MongoDB.
+* [github-actions](https://github.com/FerretDB/github-actions): This repository houses our shared GitHub Actions for FerretDB, dance, and other repositories.
+* [dance (stub)](https://github.com/FerretDB/dance): This repository is a FerretDB integration testing tool.
+
+For those taking their first steps in contributing to an open source project.
+Please take a look at this post on [how to contribute to open source software](https://www.ferretdb.io/how-to-contribute-to-open-source-2022/).
+
+## Get Started
+
+You don’t have to be a developer to contribute to FerretDB projects, you can even get started by helping us improve this documentation.
+If you have any questions or suggestions on how we can improve, kindly join our [Slack chat](https://github.com/FerretDB/FerretDB/discussions) or [GitHub discussions](https://join.slack.com/t/ferretdb/shared_invite/zt-zqe9hj8g-ZcMG3~5Cs5u9uuOPnZB8~A).
+We appreciate your feedback.
+
+## Contributing to this documentation
+
+If you find anything confusing or missing in the documentation, click the “Edit this page” link at the bottom of almost every page in the documentation.
+More information on contributing to the documentation can be found [here](https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md#contributing-documentation).
+
+## Contributing to the FerretDB repository
+
+* To contribute to this [FerretDB project](https://github.com/FerretDB/FerretDB/), please read the [CONTRIBUTING.md](https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/FerretDB/FerretDB/blob/main/CODE_OF_CONDUCT.md) guideline to know more.
+* Is everything working as it should?
+If not, please let us know by [creating an issue](https://github.com/FerretDB/FerretDB/issues/new/choose).
+You can create issues for bugs, documentation, features, or enhancements.
+* You can identify issues you would like to work on by looking at the [open issues](https://github.com/FerretDB/FerretDB/issues) for this repository.
+* For first-time contributors looking for issues to work on, kindly check the issues that are labeled [good-first-issues](https://github.com/FerretDB/FerretDB/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```

Comment:
I think we should link to https://github.com/FerretDB/FerretDB/contribute instead